Lesson3Homework/src/Application.jsx

```diff
@@ -0,0 +1,15 @@
+// package imports
+import "core-js/es6/map";
+import "core-js/es6/set";
+import "./styles/styles.scss";
+import React from "react";
+import ReactDOM from "react-dom";
+// User imports
+import Unity from "./unifiedModules.jsx";
+//
+class App extends React.Component {
+    render() {
+        return (<><Unity/></>)
```

Comment:
```suggestion
        return (<Unity/>);
```
тут смысл в <></> отпадает, так как мы возвращаем только один узел, а не два (и более) парных узла/элемента/компонента